api/server_test.go

```diff
@@ -34,12 +37,39 @@ func TestAPI(t *testing.T) {
 	sampleConfig.UsersConfiguration.MaxActiveUsers = 100
 	obj := e.POST("/create").WithJSON(sampleConfig).
 		Expect().
-		Status(http.StatusOK).JSON().Object()
-	ltId := obj.Value("loadAgentId").String().Raw()
+		Status(http.StatusCreated).JSON().Object()
+	rawMsg := obj.Value("message").String().Raw()
+	parts := strings.Split(rawMsg, ":")
+	require.Len(t, parts, 2)
+	ltId := strings.TrimSpace(parts[1])
 
 	e.POST(ltId + "/run").Expect().Status(http.StatusOK)
 	e.POST(ltId+"/user/add").WithQuery("amount", 10).Expect().Status(http.StatusOK)
 	e.POST(ltId+"/user/remove").WithQuery("amount", 3).Expect().Status(http.StatusOK)
+	e.POST(ltId+"/user/add").WithQuery("amount", 0).Expect().
```

Comment:
This is unrelated from the PR but since we are touching a lot of things I think it might be a good place to have a discussion.

I am thinking that `/loadagent/{id}/user/add` should really be `/loadagent/{id}/addusers` or even simply `/loadagent/{id}/add`. The reason is that `user` is not really an API entity. You can't get or even create a user. The only API entity is the loadagent. A `GET` to `/loadagent/{id}` should in fact return it.

@reflog, @agnivade I'd like your thoughts on this.

Reply:
fine by me :)

Reply:
I prefer `/loadagent/{id}/addusers`.

Reply:
Yes that version is definitely more precise. Sounds good to me. We can probably make that a separate ticket.

Reply:
Created - https://mattermost.atlassian.net/browse/MM-21708